writer/README.md

```diff
@@ -3,6 +3,10 @@
 See the [godoc](https://godoc.org/github.com/signalfx/signalfx-go/writer) for
 information on what they are and how to use them.
 
+> [!WARNING]  
+> `writer` package is **deprecated** and no longer supported.
```

Comment:
@akubik-splunk Can we say that it say that it is no longer supported or do we have to write something like

```suggestion
> `writer` package is **deprecated** and will reach End of Support on April 17, 2025.
```

If so then from when the time counts from? Release day?